src/targets/registry.ts

```diff
@@ -362,7 +362,11 @@ export class RegistryTarget extends BaseTarget {
       );
     }
     // Update the manifest
-    const updatedManifest = { ...packageManifest, version };
+    const updatedManifest: { [key: string]: any } = {
+      ...packageManifest,
+      version,
+    };
+    updatedManifest.createdAt = new Date().toISOString();
```

Comment:
Just to confirm: the manifest is what ends up [here](https://github.com/getsentry/sentry-release-registry/blob/master/packages/npm/%40sentry/browser/7.109.0.json), correct? If yes, then I think adding a new field should be safe

Reply:
Yes